_posts/2015-04-29-caching-external-apis.markdown

```diff
@@ -1,6 +1,6 @@
 ---
 layout:       post
-title:        Caching External API's in Rails for a Ginormous Speed Boost
+title:        Caching External APIs in Rails for a Ginormous Speed Boost
```

Comment:
Yeah. It's best not to put an apostrophe on acronyms because the rules are nasty and unclear. The one I think is most widespread is to do one if the acronym is a single letter: I got straight A's and not if it's multiple: I can sing the ABCs.